README.md

```diff
@@ -13,6 +13,12 @@
 ---
 
 ## Installation and Usage
+Before installing make sure you have the current LTS (v14~18) of Node.js. You can check your node version using ```bash node -v```. To update to latest node version run
+```bash
+sudo npm cache clean -f
+sudo npm install -g n
+sudo n stable
+```
```

Comment:
1. `sudo` should not be used because it will install packages with root privileges: https://github.com/tj/n#:~:text=lib%2C%20and%20share.-,To%20avoid%20requiring%20sudo,include%20%2Fusr%2Flocal%2Fshare,-If%20npm%20is

2. The npm cache clean step should be unnecessary. Each npm version will have it's own cache anyway.

3. Personally, I use nvm or pnpm so I don't think it's necessary to include these specific steps. We should just say the minimum Node.js version is LTS v14.

Reply:
Okay I just did a quick search on how to update node.js and found that on stack exchange

Reply:
I removed the installation commands and changed the wording of the minimum Node section version section